monitor-core/src/main/java/net/spals/appbuilder/monitor/core/TracerTag.java

```diff
@@ -0,0 +1,43 @@
+package net.spals.appbuilder.monitor.core;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import io.opentracing.Tracer;
+import net.spals.appbuilder.config.TaggedConfig;
+import org.inferred.freebuilder.FreeBuilder;
+
+/**
+ * A bean which represents a tag
+ * to be attched to a {@link Tracer} instance.
+ *
```

Comment:
Another random FWIW, if you highlight this javadoc you should actually put <p> to make the newline in javadoc rendering. And of course this would be handled via macros or the build system.